source/developers/quickstarts/setup_a_backend_dev_environment.rst

```diff
@@ -180,15 +183,15 @@ checkout:
 
 .. code-block:: bash
 
-   tutor dev run --mount=${PWD}/edx-platform lms bash
+   tutor dev run --mount=./edx-platform lms bash
 
 Now within the container, install python requirements and rebuild static assets
 for your local checkout:
 
 .. code-block:: bash
 
    pip install -e .
-   npm clean-install
+   npm install
```

Comment:
This is the only actual change. 

It's working around the `npm clean-install` issue I'm trying to fix here: https://github.com/openedx/edx-platform/pull/30883